jiant/tasks/evaluate/core.py

```diff
@@ -67,10 +67,15 @@ def __init__(self):
 
     def update(self, batch_logits, batch_loss, batch, batch_metadata):
         self.logits_list.append(batch_logits)
-        self.guid_list.append(batch_metadata.get("guid"))
+        batch_guid = batch_metadata.get("guid")
```

Comment:
Will this cause an error for tasks without `guid` in the batch_metadata?

Reply:
The `.get` returns `None` in that case (this is actually done specifically for cases of tasks without `guids`).